apps/dashboard/config/locales/en.yml

```diff
@@ -157,6 +157,7 @@ en:
     all_apps_table_sub_category_column: "Sub Category"
 
     unknown: "Unknown"
+    not_grouped: "Not Grouped"
```

Comment:
Key "not_grouped" is fine but the human friendly title I think I might prefer "Other Apps".

So if you are scrolling down the list you might see:

1. Chemistry Apps
2. Biology Apps
3. Physics Apps
4. Other Apps